test/index.js

```diff
@@ -81,9 +81,13 @@ $.def ('$26')
       (a => b => c => d => e => f => g => h => i => j => k => l => m => n => o => p => q => r => s => t => u => v => w => x => y => z =>
          [a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, r, s, t, u, v, w, x, y, z]);
 
-const reset = ((checkTypes, ...types) => () => {
-  $.config.checkTypes = checkTypes;
-  $.config.env.splice (0, Infinity, ...types);
+const test = ((checkTypes, ...types) => async (description, thunk) => {
```

Comment:
No need to take 'em variadically here

Reply:
I'm doing this to make a copy of the array. We could do this instead:

```diff
-const test = ((checkTypes, ...types) => async (description, thunk) => {
+const test = ((checkTypes, types) => async (description, thunk) => {
   try {
     await _test (description, thunk);
   } finally {
     $.config.checkTypes = checkTypes;
     $.config.env.splice (0, Infinity, ...types);
   }
-}) ($.config.checkTypes, ...$.config.env);
+}) ($.config.checkTypes, [...$.config.env]);
```

Which approach do you prefer?

Reply:
Ahhh, haha! I guess I prefer the latter, as I would've never guessed that the former was intentional!